src/app/champion-a-cause/champion-a-cause.component.html

```diff
@@ -7,13 +7,14 @@
     <option value='name'>Employee Name</option>
 </select>
 
-
 <ul class="list">
     <li class="list-item"
         *ngFor="let champion of employees | searchEmployees:'firstname,lastname,champion,title':searchText">
         <div class="list-content">
             <div class="list-title">{{champion.champion}}</div>
-            <div>{{champion.firstname}} {{champion.lastname}}</div>
+            <a [routerLink]="['/employee', champion.employeeid]"> 
```

Comment:
the default for the site is for link to not have a text-decoration.  however in this case I think the name should be underlined on hover.  Adding the style below to the champion a cause css should do the trick
```suggestion
            <a [routerLink]="['/employee', champion.employeeid]"> 
```
a:hover {
  text-decoration: underline;
}